src/assets/scss/pages/story.scss

```diff
@@ -1,11 +1,36 @@
 .author-information {
+  align-items: center;
   display: flex;
   flex-direction: row;
-  margin-left: 3em;
+  flex-wrap: wrap;
+  justify-content: flex-start;
   margin-top: 0.5em;
 
+  & > * {
+    margin-right: 4rem;
+  }
+
+  .name,
+  .target-product {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+    .product-logo {
+      height: fit-content;
+      margin: auto 5px;
+      width: 150px;
+    }
+  }
+
   .user-avatar {
     margin: 0px 0px 0px 16px;
+
+    .avatar {
+      height: auto;
+      width: 50px;
```

Comment:
let's reduce this down to 25px. We don't want to take too much space in the header as the most important are for the user to focus is the Information area